lighthouse-core/report/html/report-styles.css

```diff
@@ -143,6 +149,9 @@
     --circle-size-big: 96px;
     --circle-size: 72px;
     --header-padding: 16px 0 16px 0;
+    --plugin-badge-size: 28px;
+    --plugin-badge-size-big: 32px;
+    --plugin-icon-size: 75%;
```

Comment:
this value seems like it can be inlined in the style rule

Reply:
but it varies over mobile/desktop

Reply:
ok sg.